guardian/riff-raff
# fix: Show use of default parameter values as info

## What does this change?
Some Riff-Raff parameters have default values. Whilst it isn't necessary to have these in one's `riff-raff.yaml` file, doing so doesn't have any negative impact. A warning is too strong here, and info is better.

I can't quite determine from the revision history _why_ this was a warning; my assumption is that it creates a longer `riff-raf.yaml` file. This reasoning is solid for when this file is hand-written, as it's potentially harder to reason about, but we now have [a generator within GuCDK](https://github.com/guardian/cdk/blob/9bc15e64fa4af51224288c17edcc85aacc6a1900/src/experimental/riff-raff-yaml-file/README.md), which removes the complexities of the `riff-raff.yaml` file entirely. For projects that use the generator, since https://github.com/guardian/cdk/pull/1838, these warnings have increased.

[Deployment of the `Ariadne` project](https://riffraff.gutools.co.uk/deployment/view/00797052-54ad-4e6a-8f0b-0c6cebd2a89d) is a rather extreme example of the warnings:

![image](https://github.com/guardian/riff-raff/assets/836140/5b5983fd-e9cc-44e9-97cf-4974fa2883fa)

## How to test
- Deploy to CODE
- Using CODE, deploy the `Ariadne` project, and observe a reduction in the number of warnings
